docs/source/release/v6.10.0/SANS/Bugfixes/37219.rst

```diff
@@ -0,0 +1 @@
+- An error is no longer produced when saving from the ``Save Other`` dialog.
```

Comment:
Would it be worth linking to the documentation (https://docs.mantidproject.org/nightly/interfaces/isis_sans/Runs%20Tab.html#save-options), or referring to the SANS GUI?